index.js

```diff
@@ -0,0 +1,39 @@
+const yaml = require('js-yaml');
+
+module.exports = robot => {
+    robot.on('pull_request.opened', async context => {
+        const owner = context.payload.repository.owner.login;
+        const repo = context.payload.repository.name;
+        const number = context.payload.pull_request.number;
+        let config;
+
+        const files = await context.github.pullRequests.getFiles({ owner, repo, number });
```

Comment:
similar to above. You can replace this with 
`context.github.pullRequests.getFiles(context.issue());`

https://probot.github.io/probot/latest/Context.html#issue